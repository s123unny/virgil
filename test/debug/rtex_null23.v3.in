b ../core/rtex_null23.v3 7
run
bt
info l
c
q
