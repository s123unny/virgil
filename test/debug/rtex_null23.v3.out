breakpoint|../core/rtex_null23.v3 7|0
stop|stopOnBreakpoint
bt|main|../core/rtex_null23.v3|7
variable|0|z|false|bool|false
variable|1|a|null|rtex_null23a|false
variable|2|b|null|rtex_null23b|false
variable|3|x|null|rtex_null23a|false
stop|end
result|!NullCheckException
	in main() [../core/rtex_null23.v3 @ 7:17]

